app/views/topics/show.html.erb

```diff
@@ -8,17 +8,7 @@
   <div class="col-md-8">
     <p class="lead"><%= @topic.description %></p>
```

Comment:
You can just embed the topic partial right here, passing `@topic` as local variable `topic`.  It should look just like [this](https://github.com/ffquint/bloccit/blob/0d7322fea9e49d24b46df2a86e690dfaa8f79ff4/app/views/topics/index.html.erb#L6) but without the loop (since there is only a single topic)

This will change the look a bit, as you'll now have the name of the topic and a link to itself (the page you are already on) which is a bit odd, but worth considering.